src/config/section/network.js

```diff
@@ -186,14 +187,20 @@ export default {
       columns: ['ipaddress', 'state', 'associatednetworkname', 'virtualmachinename', 'allocated', 'account', 'zonename'],
       details: ['ipaddress', 'id', 'associatednetworkname', 'virtualmachinename', 'networkid', 'issourcenat', 'isstaticnat', 'virtualmachinename', 'vmipaddress', 'vlan', 'allocated', 'account', 'zonename'],
       tabs: [{
-        name: 'configure',
-        component: () => import('@/views/network/IpConfigure.vue')
+        name: 'details',
+        component: () => import('@/components/view/DetailsTab.vue')
+      }, {
+        name: 'Firewall',
+        component: () => import('@/views/network/FirewallRules.vue')
+      }, {
+        name: 'Port Forwarding',
+        component: () => import('@/views/network/PortForwarding.vue')
+      }, {
+        name: 'Load Balancing',
```

Comment:
@RitchieVincent add the `show()` methods for all these tabs, as discussed and described earlier see the individual issues tickets on what condition to check (for example, 'Firewall' in case of Firewall tab etc).